ndmg/scripts/ndmg_bids.py

```diff
@@ -504,34 +425,23 @@ def main():
     outDir = result.output_dir
     subj = result.participant_label
     sesh = result.session_label
-    task = result.task_label
     run = result.run_label
     buck = result.bucket
     remo = result.remote_path
     push = result.push_data
-    stc = result.stc
     debug = result.debug
-    modality = result.modality
     seeds = result.seeds
     skipeddy = result.sked
     skipreg = result.skreg
     clean = result.clean
     vox_size = result.vox
-    minimal = result.minimal
-    log = result.log
     atlas_select = result.atlas
-    dataset = result.dataset
-    hemi = result.hemispheres
     mod_type = result.mod
     track_type = result.tt
     mod_func = result.mf
     reg_style = result.sp
     modif = result.modif
 
-    # Raise NotImplementedError if functional analysis is
-    if modality=='func':
```

Comment:
I would actually be a fan of keeping the --modality flag, having it raise NotImplementedError if func, and defaulting to DWI - because eventually we are going to add fMRI back in as a call to CPAC

Although actually if we do it the way we were talking about in slack, this wouldn't involve the ndmg codebase at all, so maybe this is a good idea, and it's way cleaner to remove the flag. Fine with whatever we end up deciding to do.